jupyter_elfinder/elfinder.py

```diff
@@ -102,7 +102,7 @@ class connector:
         "ogg": "application/ogg",
         "7z": "application/x-7z-compressed",
         # video
-        "ogm": "appllication/ogm",
+        "ogm": "application/ogm",
```

Comment:
I think this might solve a bug.

Reply:
Yes, it should fix ogm video, sharp eye!